includes/Admin/Activation_Notice.php

```diff
@@ -97,12 +107,9 @@ public function enqueue_assets( $hook_suffix ) {
 		 */
 		unset( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification, WordPress.VIP.SuperGlobalInputUsage
 
-		wp_enqueue_style(
-			'web-stories-activation-notice-roboto',
-			'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,900&display=swap',
-			[],
-			WEBSTORIES_VERSION
-		);
+		$this->register_font->register();
+		$font_handle = $this->register_font->get_handle();
+		wp_enqueue_style( $font_handle );
```

Comment:
Really can't wait for #7386

Reply:
I posted my comment here - https://github.com/google/web-stories-wp/issues/7386#issuecomment-836691127

We already have to convert the two other places that use this class. One more place isn't going to hurt that much.